docs/Demo.md

```diff
@@ -12,37 +8,137 @@
 
 In this demo, you will get a taste of how Pelorus captures a change going through the application's delivery cycle.
 
-1. Initializing Pelorus sets the baseline by looking at existing stored data
-2. Create a new commit
-3. Watch as the metrics and trends change as new versions roll out
+* Install and configure Pelorus
+* Install a sample application that Pelorus will measure
+* Set a baseline of data for Pelorus measurements
+* Create a new commits, and Github issues
+* Watch as the metrics and trends change as new versions roll out
 
-Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
+Understand that Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
 
-## Prerequisites
+--------------------------
 
-* Pelorus is installed and running
-  * Follow the steps outlined in the [todolist install guide](../samples/todolist.md#pelorus-install-steps)
+# Lead Time for Change and Deployment Frequency
 
-* The todolist sample application is installed and running
-  * Follow the steps outlined [todolist sample install guide](../samples/todolist.md#todolist-sample-application-install-steps)
+![dora_lead_deployment](img/dora_metrics1.png)
 
-> **Note:**
-> Ensure that your github fork of mig-demo-apps is correctly set as the uri value in the todolist BuildConfig.
-```
-- kind: BuildConfig
-  apiVersion: build.openshift.io/v1
-  metadata:
-    name: todolist
-<snip>
-      source:
-        type: Git
-        git:
-          uri: https://github.com/weshayutin/mig-demo-apps.git
-          ref: master
+## Workflow - Install
+- install and configure Pelorus
+- install a sample application
 
-```
+### Pelorus install steps
+* Create the pelorus namespace
+
+        oc create namespace pelorus
+
+* Install the required granfa and prometheus operators
+
+        helm install operators charts/operators --namespace pelorus
+    
```

Comment:
Wait for the grafana-operator-controller-manager and prometheus-operator pods to be Running, similar output to:

```
$ oc get pods -n pelorus
NAME                                                   READY     STATUS    RESTARTS   AGE
grafana-operator-controller-manager-7678cc5c7c-spvls   2/2       Running   0          22s
prometheus-operator-559d659944-fvsjg                   1/1       Running   0          10s

```